dotnet-desktop-guide/framework/winforms/controls/datagrid-control-overview-windows-forms.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: "DataGrid Control Overview"
+description: Learn about this overview of the DataGrid control in Windows forms, which is retained for both backward compatibility and future use.
```

Comment:
same note as before. Also, please add something from the first content paragraph, not the note. Perhaps something related to `displays data in a series of rows and columns.`